sql/executor.go

```diff
@@ -146,6 +153,12 @@ func (e *Executor) execStmts(sql string, params parameters, planMaker *planner)
 			result = makeResultFromError(planMaker, err)
 		}
 		resp.Results = append(resp.Results, result)
+		// TODO(pmattis): Is this the correct time to be releasing leases acquired
+		// during execution of the statement?
+		//
+		// TODO(pmattis): Need to record the leases used by a transaction within
+		// the transaction state and restore it when the transaction is restored.
+		planMaker.releaseLeases(e.db)
```

Comment:
I'm not positive this is the correct place to release. See the TODO at line 156.